README.md

```diff
@@ -13,3 +13,20 @@ The library provides C++ classes that manage three tasks:
  - Triggering digitized hits
     - A simple algorithm that counts number of digitized hits falling in a sliding time winodw
     - Selectable trigger window and thershold
+
+## WCTE/WCSim usage
+Simple runtime procedures. First set up your ROOT and WCSIM:
+```
+source your_thisroot.sh
+export WCSIMDIR=your_WCSIM_installation
+```
+Then set up the MDT environment.
+```
+source envMDT.sh
```

Comment:
According to L38 in `envMDT.sh`, this assumes you've `cd`ed into MDT folder already?

Reply:
resolved inside `envMDT.sh`